### PR TITLE
bgrzesik-dev to main

### PR DESCRIPTION
**Pull request by bgrzesik**

### CODE REVIEW

**Comment by bgrzesik** [comment]
Gentle ping 😅

**Comment by bgrzesik** [comment]
I have a draft of `main.py` with `RequestRouter` but I am unable to test it on the raspberry in the coffee shop 😅 Please start the review and I will confirm that the `main.py` works in few hours.

File: server/comm/presentation/StreamParser.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
 from server.comm.presentation.message_utils import *
```
Comment (bzapa) [inline server/comm/presentation/StreamParser.py]: I think this class can be removed

File: server/comm/presentation/message_utils.py [hunk 2]
```diff
@@ -14,8 +14,9 @@
```
Comment (bzapa) [inline server/comm/presentation/message_utils.py]: Is this class used?

**Review by bzapa — APPROVED** [review]
Please remove unused code or mark as deprecated

**Pull request merged**